gdsfactory/components/coupler_adiabatic.py

```diff
@@ -1,12 +1,42 @@
-from typing import Tuple
-
-import picwriter.components as pc
+from gdspy import Path
```

Comment:
why not use the `gf.components.bend_s` from the generic pdk?

Reply:
Since the taper sections do not have the same widths on both ends, the s-bends need to be tapered as well (that was how it was it in the picwriter component too). I am not sure if that is doable with the `bend_s` component.